nexus/src/external_api/http_entrypoints.rs

```diff
@@ -3756,15 +4044,3 @@ async fn session_sshkey_delete(
     };
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
-
```

Comment:
What's up with the removal of this test?

Reply:
I'm not sure how that happened exactly. Good catch.

Reply:
Fixed now.